tidb-cloud/migration-job.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Migrate Online or On-Premises Data Using Migration Jobs
+summary: Learn how to create migration jobs to migrate online or on-premises data to TiDB Cloud.
+---
+
+# Migrate Online or On-Premises Data Using Migration Jobs
+
+This document describes how to use a migration job to migrate online or on-premises data to TiDB Cloud.
+
+You can import data in CSV, Parquet, SQL, and Aurora Snapshot formats to TiDB Cloud, as described in [Migration Overview](/tidb-cloud/tidb-cloud-migration-overview.md). Many of these scenarios are offline data import scenarios. When importing such data, to ensure the consistency of upstream and downstream data, you need to stop the upstream business, then export the offline files from the database, and then import the data to TiDB Cloud. It will bring a long downtime to the business, which is unacceptable in some cases.
+
+To solve this problem, TiDB Cloud provides the migration job feature, which supports full migration and incremental migration, allowing you to migrate your business from the upstream database to TiDB Cloud within a short downtime window.
+
+A migration job supports the following data sources:
+
+- MySQL 5.6-8.0 on premises or on a cloud
+- AWS Aurora MySQL 5.6 and 5.7
+- AWS RDS MySQL 5.7
+
+A migration job supports the following network types:
+
+- Public IP
+- Private Link
+
+A migration job supports data migration within the same region and cross regions.
+
+## Limitations
+
+- The system databases including `mysql`, `information_schema`, `information_schema` and `sys` will be filtered out during migration (even when you migrate the entire upstream MySQL instance), and will not be migrated to TiDB Cloud.
```

Comment:
in first release, we only deploy on `us-west-2` region and eks provider. that means old clusters created using aws provider will not see `data migration` tab.